great-idea-website/css/index.css

```diff
@@ -65,66 +65,107 @@ h1, h2, h3, h4, h5 {
 
 /* Your code starts here! */
 
-main {
+.main-content-area {
 	width: 900px;
 	display: block;
 	margin: auto;
+	margin-top: 20px;
 }
-header {
-	height: 80px;
-	padding: 5px;
+
+/* Main Navigation Styling */
+
+.main-nav {
 	margin: auto;
-	vertical-align: middle;
+	/*text-align: center;*/
+	width: 900px;
+	height: 100px;
+	position: relative;
```

Comment:
I don't think you need the position : relative here.  It would be useful if you wanted to offset from it's static position in the document flow. but you're not adjusting the relative positioning using top/bottom/left/right, so it's not necessary.